docs/commands/cd.md

```diff
@@ -21,5 +21,13 @@ If you didn't already know, the `cd` command is very simple. It stands for 'chan
 /home/username/Desktop/super/duper/crazy/nested/folders> cd
 /home/username> cd ../../usr
 /usr> cd
-/home/username> 
+/home/username>
+```
+
+Using `cd -` will take you to the previous directory:
```

Comment:
Nice touch. Could you add similar descriptions to the previous example blocks as well?